css/style.css

```diff
@@ -0,0 +1,1248 @@
+  }
+
+  .portfolio-container {
+    position: relative;
+    -webkit-flex-direction: row;
+    flex-direction: row;
+    -webkit-flex-wrap: wrap;
+    flex-wrap: wrap;
+    -webkit-align-items: center;
+    align-items: center;
+    -webkit-justify-content: space-around;
+    justify-content: space-around;
+    padding-top: 2em;
+  }
+}
+
+
+
+/*
+PORTFOLIO SLIDER
+*/
+
+.portfolio-slider {
+  width: 100%;
+}
+
+.portfolio-slider-figure {
+  position: relative;
+  overflow: hidden;
+  direction: rtl;
+}
+
+.portfolio-slider-figure img {
+  min-width: 40em;
+  margin: 0 auto;
+}
+
+.portfolio-slider-overlay {
+  opacity: .7;
+}
+
+.portfolio-slider-caption {
+  position: absolute;
+  top: 10%;
+  right: 18%;
+  width: 14em;
+  line-height: 1.1;
+  text-align: left;
+  text-shadow: 1px 1px 5px #2e2e2e;
+}
+
+.portfolio-slider-caption p {
+  font-size: 1.333em;
+}
+
+.portfolio-slider-button {
+  padding: 1em 1em;
+  margin: .8em 0;
+  background-image: linear-gradient(#4dbff7, #5087c8);
+}
+
+.portfolio-slider-button:hover {
+  cursor: pointer;
+}
+
+.portfolio-slider-button:before,
+.portfolio-slider-button:after {
+  font-family: "ico-fonts";
+}
+
+
+.portfolio-slider-button:after {
+  content: "\e60a";
+  padding-left: 1.333em;
+}
+
+.portfolio-slider-button:before {
+  content: "\e610";
+  padding-right: 1em;
+  color: rgba(0,0,0,.3);
+}
+
+.portfolio-slider-bullets[aria-hidden="true"] {
+  visibility: hidden;
+}
+
+@media screen and (min-width: 50em) {
+
+  .portfolio-slider-caption {
+    top: 20%;
+    right: 25%;
+  }
+
+  .portfolio-slider-bullets[aria-hidden="true"] {
+    visibility: visible;
+  }
+
+  .portfolio-slider-bullets {
+    padding: .3em;
+    font-size: 1.777em;
+    text-align: center;
+  }
+
+  .portfolio-slider-bullets li {
+    display: inline;
+  }
+
+  [title$="bullet"]:before {
+    content: "•";
+    color: #d4d4d4;
+  }
+
+  .slider-bullet-active:before {
+    color: #fd6632;
+  }
+}
+
+
+
+/*
+PORTFOLIO NAVIGATION
+*/
+
+[title="portfolio-categories"] li {
+  display: block;
+  margin-top: .9em;
+  margin-bottom: .5em;
+  text-align: center;
+}
+
+.portfolio-menu-item {
+  padding: .3em;
+  font-size: 1.333rem;
+  text-decoration: none;
+  color: #d4d4d4;
+}
+
+.portfolio-selected-category {
+  color: #fff;
+  background-color: #fd6632;
+}
+
+@media screen and (min-width: 50em) {
+
+
+  [title="portfolio navigation"] {
+    display: -webkit-flex;
+    display: flex;
+    -webkit-flex-basis: 100%;
+    flex-basis: 100%;
+    -webkit-justify-content: center;
+    justify-content: center;
+    margin-top: 1em;
+    margin-bottom: 1.777em;
+  }
+
+  [title="portfolio-categories"] li {
+    display: inline-block;
+  }
+
+  .portfolio-menu-item {
+    font-size: 1rem;
+  }
+}
+
+
+/*
+PORTFOLIO ITEMS
+*/
+
+.portfolio-item {
+  position: relative;
+  max-width: 300px;
+  margin-top: .5em;
+  margin-bottom: 2em;
+  text-align: center;
+}
+
+.portfolio-opacity {
+  opacity: .7;
+}
+
+.portfolio-link-hover:hover {
```

Comment:
I'd probably name this `.portfolio-link` rather than `.portfolio-link-hover` -- at some point, it might be displayed differently, but in the immediate case, the use of `-hover` implies that this is a modifier class for `.portfolio-link` (which it isn't). Additionally, `.portfolio-link-hover:hover` ends up being redundant.

Your class name should describe what the object is or the state that it's currently in, not how it's implemented.